packages/core-js/utils/ethers/events.js

```diff
@@ -15,7 +15,7 @@ function findEvent({ receipt, eventName, contract = undefined }) {
 
   events = events.filter((e) => e.event === eventName);
   if (!events || events.length === 0) {
```

Comment:
This is a tool and should not throw IMO. Throwing, warning, etc, should be in charge of the user of the tool.

In my case, I needed to assert that an event was not found in a receipt.